lib/engine/game/g_18_fl/game.rb

```diff
@@ -312,7 +312,7 @@ class Game < Game::Base
               hexes: [],
               tiles: %w[3 4 6o 6fl 8 9 58],
               closed_when_used_up: true,
-              when: %w[track owning_player_or_turn],
+              when: %w[track owning_player_track],
             },
```

Comment:
the ability grants an extra tile lay. With the fix to crop.all_abilities an operator is prompt at the last operating step (after track and train buy) if they want to use the ability. Changed to the correct timing for owning player.